006_buildkite_importing_pipelines/README.md

```diff
@@ -0,0 +1,374 @@
+# Driving Buildkite Pipelines with CUE
+<sup>by [Jonathan Matthews](https://jonathanmatthews.com)</sup>
+
+This guide explains how to convert Buildkite pipeline files from YAML to CUE,
```

Comment:
Is there a more precise name for what type of pipeline we are talking about here? Given our conversation yesterday?

Reply:
Done.

Reply:
Happy with that. One way to help here is to link to docs to confirm which one we are talking about (even if those docs don't actually use a specific name)